src/test/resources/features/users/delete_users.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Delete Users  
+
+  In order to delete an account
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+
+  @functional
+  @createUser
+ Scenario: Verify that is possible to delete an user
+    When I send a GET request to "/user/delete/{id}"
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "board/boardSchema.json" JSON schema
+    And I validate that the response contains the following values
+      | status  | 200                                                    |
+      | message | "User deleted" |
+
+  @negative
+  @createUser
```

Comment:
Please put tags in a single line, first the corresponding ones for categorization, after the corresponding ones to preconditions or posconditions.
`@functional @createUser`

Reply:
Thanks, It's done.